### DISCUSSION TITLE
add option to enable right click for websites that block it

### DISCUSSION THREAD

**Discussion by shalva97** [minbrowser/min#1464] [Ideas]
some websites completely disable right click, for example you cannot right click on anything on this [website](https://www.pngitem.com/middle/TTixhxo_two-green-leaves-png-transparent-png/)

Firefox has a work around, if you use shift+click it still shows you the context menu, even if it is replaced with something else. 

Alternatives? well, disable the API to that disables right clicks probably... or make a setting... Im not sure but some prefix+click shortcut would be perfect

**Comment by Milo123459** [comment]
This is probably going to be quite hard as we'd have to re-work right click kind-of (from my understanding) allowing you to open up the right click menu, but I'm really not the one to ask
